implementation-status.md

```diff
@@ -32,13 +30,18 @@ Tip: Chrome channel releases are tracked at [https://googlechromelabs.github.io/
 * [Issue 854935](http://crbug.com/854935): Closed captions in Picture-in-Picture are not supported.
 
 # Microsoft Edge
-https://dev.windows.com/en-us/microsoft-edge/platform/status/pictureinpicture
+Under consideration.
+https://developer.microsoft.com/en-us/microsoft-edge/platform/status/pictureinpicture/
```

Comment:
Edgium has the feature.

Reply:
@MarionDaly, can you confirm? Edge planning to ship this too?

Reply:
Edge Canary channel has Picture-in-Picture API enabled as you can see.
![image](https://user-images.githubusercontent.com/634478/61215160-43c4e700-a70a-11e9-8dfa-0b1ccf0e14b1.png)

Reply:
FYI I've opened https://github.com/WICG/picture-in-picture/pull/153